file-formats/srvc/type/zif_aff_srvc_v1.intf.abap

```diff
@@ -0,0 +1,75 @@
+INTERFACE zif_aff_srvc_v1
+  PUBLIC.
+  "! <p class="shorttext">Consumer Type</p>
+  "! Consumer type
+  "! $values {@link zif_aff_srvc_v1.data:co_cons_type}
+  TYPES ty_cons_type TYPE c LENGTH 2.
+
+  CONSTANTS:
+    BEGIN OF co_cons_type,
+      "! <p class="shorttext">OData</p>
+      "! OData
+      "! $enumValue 'OData'
+      odata       TYPE ty_cons_type VALUE '01',
+      "! <p class="shorttext">Web Service</p>
+      "! Web service
+      "! $enumValue 'WebService'
```

Comment:
I guess I Would remove the `$enumValue` here, but keek it for "OData" and "RFC".

Reply:
Done